source/code-snippets/usage-examples/replaceOne.js

```diff
@@ -0,0 +1,51 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+// Replace the following with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const collection = database.collection("movies");
+
+    // create a filter for a movie to update
+    const filter = { title: "Blacksmith Scene" };
+    const options = {
+      // create a document if no documents match the filter
+      upsert: true,
+    };
+    // create a new document that will be used to replace the existing document
+    const replacement = {
+      title: "Sandcastles in the Sand",
+      plot:
+        "Robin Sparkles mourns for a relationship with a mall rat at an idyllic beach.",
+    };
+
+    const result = await collection.replaceOne(filter, replacement, options);
+
+    if (result.modifiedCount === 0 && result.upsertedCount === 0) {
+      console.log("No changes made to the collection.");
+    } else {
+      if (result.matchedCount === 1) {
+        console.log("Matched " + result.matchedCount + " documents.");
+      }
+      if (result.modifiedCount === 1) {
+        console.log("Updated one document.");
+      }
+      if (result.upsertedCount === 1) {
+        console.log(
+          "Inserted one new document with an _id of " + result.upsertedId._id,
```

Comment:
Question: Is this comma necessary/expected?

Reply:
An odd artifact of our prettier, it seems. Not much I can do about it.

Reply:
I wonder if that is an older version. I was able to run eslint with react-app and prettier configs and it did not suggest/add a comma there.